cou/steps/ceph.py

```diff
@@ -266,3 +266,57 @@ async def set_require_osd_release_option(model: Model, apps: Sequence[Applicatio
             continue
 
         await set_require_osd_release_option_on_unit(model, ceph_mon_unit_name)
+
+
+def get_mon_apps(apps: Sequence[Application]) -> Sequence[Application]:
```

Comment:
this can be either inline or used by other functions e.g. `set_require_osd_release_option` also gets ceph_mon apps

Reply:
Yes, I thought this function might be useful in other functions in the future, but that's also a refactoring that's unrelated to this change. :)